packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -409,6 +409,41 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
     });
   });
 
+  const handleKeyDown = (event) => {
+    const { target } = event;
+    // Keyboard navigation assumes that [role="tab"] are siblings
+    // though we might warn in the future about nested, interactive elements
+    // as a a11y violation
+    const role = target.getAttribute('role');
+    if (role !== 'tab') {
+      return;
+    }
+
+    let newFocusTarget = null;
+    const previousItemKey = orientation === 'horizontal' ? 'ArrowLeft' : 'ArrowUp';
```

Comment:
Just a thought. RTL will affect this right?

Reply:
If it's the case, then we need to handle it too for the slider: https://material-ui.com/components/slider/#continuous-sliders. Looking at the Rating, it seems that we should.

Reply:
Opening an issue

Reply:
#20785.

Reply:
Never thought about that one. Actually I haven't noticed this in any a11y ressource regarding keyboard navigation.

Reply:
Actually, can we leave in the test but hold off with the implementation? It make sense for me visually but I have no idea if it makes sense that right means previous in a right-to-left language.

Reply:
E.g. in math you don't mirror the number line (which relates to the slider). But tabs have different semantics. Either way I'd be more comfortable if we have some explainer for this behavior.

Reply:
Sounds good. :)

Reply:
> The most important icons for mirroring are back and forward buttons. Back and forward navigation buttons are reversed.

-- https://material.io/design/usability/bidirectionality.html#mirroring-elements

Seems like for tabs it does make almost always sense to reverse keyboard navigation.